pdr_backend/lake/duckdb_data_store.py

```diff
@@ -159,6 +163,47 @@ def insert_from_csv(self, table_name: str, csv_folder_path: str):
             df = pl.read_csv(csv_file)
             self.insert_from_df(df, table_name)
 
+    @enforce_types
+    def move_table_data(self, from_table, to_table):
+        """
+        Move the table data from one table to another.
+        @arguments:
+            from_table - where we'll be taking data from
+            to_table - wehere we'll be inserting data
+        @example:
+            move_table_data("temp_people", "people")
+        """
+
+        # Check if the table exists
+        move_table_query = self.get_query_move_table_data(from_table, to_table)
+        self.execute_sql(move_table_query)
+
+    @enforce_types
+    def update_data(self, df: pl.DataFrame, table_name: str, column_name: str):
```

Comment:
are these intentional additions? I remember removing something very similar, specifically update_data not being used anywhere. Let's discuss this after we clear if they are intentional additions or conflict artifacts.

Reply:
same for "move_table_data"

Reply:
yeah, they are intentional implementations
- update_data is used for incremental updates as we want to modify existing records
- same with move_table_data, it's used in the swap strategy

I'll review the CRUD wrapper to make sure the functionality isn't duplicated

Reply:
yes, these are intentional additions

"remember removing something very similar, specifically update_data not being used anywhere "
can you show me? perhaps this was added ahead of time of time, and even I myself removed it as it was not being used by any update routine... now it is.

"same for "move_table_data"
same statement as above. it's now being used to do an update/swap routine.